web/rootfs/defaults/meet.conf

```diff
@@ -5,6 +5,7 @@
 {{ $ENABLE_SUBDOMAINS := .Env.ENABLE_SUBDOMAINS | default "true" | toBool -}}
 {{ $XMPP_DOMAIN := .Env.XMPP_DOMAIN | default "meet.jitsi" -}}
 {{ $XMPP_BOSH_URL_BASE := .Env.XMPP_BOSH_URL_BASE | default "http://xmpp.meet.jitsi:5280" -}}
+{{ $WHITEBOARD_URL_BASE := .Env.WHITEBOARD_URL_BASE | default "http://whiteboard.meet.jitsi" -}}
```

Comment:
Don't set a default, otherwise it will always be enabled.

At any rate you are not using this variable.